dev/badges.md

```diff
@@ -57,6 +57,7 @@ These badges will be implementable by PRs from the community nearer the time of
 
 A collection of unsorted ideas. Move one of these into "To Be Implemented" if you'd like to see it end up in the app:
 
+- Solve two-bucket in five languages: name ("Decabucket", "Piles o' Pails")
```

Comment:
Happy for this to be put into "To be implemented" with the "Piles o' Pails" name :)